crates/rethnet/tests/cli.rs

```diff
@@ -154,14 +149,13 @@ async fn node() -> Result<(), Box<dyn std::error::Error>> {
 
     // assert that the standard output of the server process contains the expected log entries:
     Assert::new(output.clone()).stdout(contains("Listening on 127.0.0.1:8549"));
-    let context = Secp256k1::signing_only();
-    for (i, default_private_key) in rethnet_defaults::PRIVATE_KEYS.to_vec().iter().enumerate() {
+    for (i, default_private_key) in rethnet_defaults::SECRET_KEYS.to_vec().iter().enumerate() {
```

Comment:
Still a mention of `default_private_key`

Reply:
Thanks, looks like I missed some. I think I got them all now in [e3ec175](https://github.com/NomicFoundation/hardhat/pull/4465/commits/e3ec175495c36b67645a5e9aba0d3151eb910729)